signac/core/jsoncollection.py

```diff
@@ -15,50 +15,60 @@
 from .synced_collection import SyncedCollection
 from .syncedattrdict import SyncedAttrDict
 from .synced_list import SyncedList
+from .synced_collection import _get_filemetadata
 
 
 class JSONCollection(SyncedCollection):
     """Implement sync and load using a JSON back end."""
 
     backend = 'JSON'  # type: ignore
 
-    def __init__(self, filename=None, write_concern=False, **kwargs):
-        self._filename = os.path.realpath(filename) if filename is not None else None
-        self._write_concern = write_concern
+    def __init__(self, filename=None, write_concern=False, suspend_sync=False, **kwargs):
         super().__init__(**kwargs)
+        self.backend_kwargs['filename'] = None if filename is None else os.path.realpath(filename)
+        self.backend_kwargs['write_concern'] = write_concern
+        self.backend_kwargs['backend'] = self.backend
+        self._id = self.backend_kwargs['filename']
```

Comment:
I tried to create `_id` as https://github.com/glotzerlab/signac/issues/249#issuecomment-592080082. Please guide.